clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiAdapter.groovy

```diff
@@ -204,6 +213,22 @@ class KubernetesClientApiAdapter {
     }
   }
 
+  boolean hardDestroyStatefulSet(String name, String namespace) {
+    exceptionWrapper("statefulSets.delete", "Hard Destroy Stateful Set $name", namespace) {
+      V1DeleteOptions deleteOptions = new V1DeleteOptions()
```

Comment:
Anything you pass as a parameter I would set in these `V1DeleteOptions` as well, since future versions of k8s are moving towards those.